### DISCUSSION TITLE
Access Forbidden on logout attempt

### DISCUSSION THREAD

**Discussion by linkp** [kanboard/kanboard#5594] [Q&A] [answered]
I just updated from 1.2.40 to 1.2.42 (skipping 1.2.41) on Debian 12.8 with PHP 8.3.13 fpm-fcgi and mysql 8.0.40. 

Selecting logout from the menu results in a 403 response and an Access Forbidden message. Additionally, the user is not logged out.

I recall similar behavior about a year ago that seems to align with #5308, although that issue was returning HTTP 500, not HTTP 403.

Logout was working fine prior to my Kanboard version update today. This affects both local and LDAP accounts.

Commenting out line 61 in `app/Controller/AuthController.php` (which was introduced  in 1.2.41 by f1470de6a09eefe3c0dc016ef11b46bb035b92a9) allows the logout to succeed.

Edited to add additional detail and a workaround.

**Comment by fguillot** [comment]
Make sure to update your plugins if you have any. Some plugins are not yet updated by their authors.

See https://kanboard.discourse.group/t/kanboard-1-2-41/3289/11

**Comment by linkp** [comment, marked as answer]
Thanks.

Updating the Customizer plugin to use creecros/Customizer#154 from @alfredbuehler allowed me to remove the comment in front of line 61 in `app/Controller/AuthController.php`.
